### DISCUSSION TITLE
Add a Dagger integration section and cleanup Ecosystem CICD docs page

### DISCUSSION THREAD

**Discussion by jpadams** [aquasecurity/trivy#5607] [Documentation]
### Description

There is a Dagger integration for Trivy that we want to share through the docs page, so I made an update. In the course of updating the page, I noticed that the Azure DevOps content was stuck in the middle of the GitHub Actions content, so I moved that out to stand alone. I also noticed that integrations were using H2 (##) markdown, but the Buildkite entry was using H3 (###), so I fixed that to make things consistent.

### Link

https://github.com/aquasec/trivy/blob/main/docs/ecosystem/cicd.md

### Suggestions

https://github.com/jpadams/trivy/blob/cicd-md-cleanup-add-dagger/docs/ecosystem/cicd.md

**Comment by itaysk** [comment]
thanks for adding it!

**Comment by jpadams** [comment]
@AnaisUrlichs and @knqyf263

Is a manual deploy of these changes required?
https://github.com/aquasecurity/trivy/pull/5608

**Reply by knqyf263** [reply]
You can see the change in the `dev` version, not `latest`.
https://aquasecurity.github.io/trivy/dev/ecosystem/cicd/

It is not versioned until the next version is released.
